### DISCUSSION TITLE
Taking Volunteers to Present for the Summer of 2022.

### DISCUSSION THREAD

**Discussion by ADParedes** [Open-Research-Program/HCOE#3] [General]
[Let me know what week you'd like to get feedback on a presentation.](https://docs.google.com/document/d/1rngg7e4HVEXw4uiU647gzeSPZc21SPUauobwyNDqa0Q/edit) 
Optional: Let me know what you are looking to work on, and what kind of feedback you are hoping to get on.

**Comment by mizquierdo2** [comment]
I am interested in presenting during the week of July 10th or July 24th. Thank you! I appreciate comments on presentation style, delivery, and how my audience was able to understand my personal speaking aims.

**Reply by ADParedes** [reply]
Great, I'll slot you in for the 24th of July! I'll follow up via email shortly with some more guidance.

**Comment by jasson907** [comment]
I can present the week of July 17th or 24th

**Reply by ADParedes** [reply]
Great, I'll slot you in for the 17th of July! I'll follow up via email with some more guidance shortly.

**Comment by spulido1** [comment]
I am interested in presenting July 20th if that is still okay.

**Reply by ADParedes** [reply]
Excellent you have been slotted to present on July 20th! I'll follow up via email with more guidance shortly.

**Comment by jaq-serrano** [comment]
I can present during the August 3rd meeting!

**Reply by ADParedes** [reply]
Awesome! You have been slotted to present on August 3rd! I'll follow up via email with more guidance shortly.
